app/jobs/asset_cleanup_job.rb

```diff
@@ -2,33 +2,39 @@
 
 class AssetCleanupJob < ApplicationJob
   def perform
-    active_revision_ids = Edition
-      .where(current: true)
-      .or(Edition.where(live: true))
-      .pluck(:revision_id)
-
-    active_file_attachment_asset_ids = FileAttachment::Revision
-      .joins(:revisions, blob_revision: [:asset])
-      .where("revisions.id" => active_revision_ids)
-      .pluck("file_attachment_assets.id")
+    clean_image_assets
+    clean_file_attachment_assets
+  end
 
-    active_image_asset_ids = Image::Revision
-      .joins(:revisions, blob_revision: [:assets])
-      .where("revisions.id" => active_revision_ids)
-      .pluck("image_assets.id")
+private
 
-    FileAttachment::Asset
-      .where.not(id: active_file_attachment_asset_ids)
-      .where.not(state: "absent")
-      .each { |asset| delete_asset(asset) }
+  def clean_image_assets
+    image_asset_used =
```

Comment:
Should this be `image_assets_used` instead of `image_asset_used` as this is potentially concerning multiple assets?

Reply:
This is looking for whether a single asset is used, it is used on each DB record.

Reply:
Ah doh, nm then.